closure-typing/storage.js

```diff
@@ -1,5 +1,28 @@
 function storage() {
+    var storage = {};
 
+    function store (inputKey, inputValue) {
+        if(arguments.length === 1){
+            var value = storage[inputKey];
+            if(value)
+                return value;
+            else
+                return 'Not found';
+        } else if(arguments.length === 2){
+            storage[inputKey] = inputValue;
+        }
+
+
+
+        return storage;
+    }
+
+    store.clear = function () {
+        storage = {};
+        return storage;
+    };
+
+    return store;
 }
 
 module.exports = storage;
```

Comment:
Good solutions, there are few problems though
Try to look at the styleguide I sent you in chat, and clean up your code a bit, also look throw String methods, there are a lot of helpers you might find useful, other then substring and indexOf
Overall - great work!